### PR TITLE
Add flutter desktop support

### PR DESCRIPTION
**Pull request by logist322**
## Synopsis

Improve `Jason`  for working under `desktops` too.

## Checklist

- Created PR:
    - [x] In [draft mode][l:1]
    - [x] Name contains `Draft: ` prefix
    - [x] Name contains issue reference
    - [x] Has `k::` labels applied
    - [x] Has assignee
- [x] Documentation is updated (if required)
- [x] Tests are updated (if required)
- [x] Changes conform code style
- [x] CHANGELOG entry is added (if required)
- [x] FCM (final commit message) is posted
    - [x] and approved
- [x] [Review][l:2] is completed and changes are approved
- Before merge:
    - [x] Milestone is set
    - [x] PR's name and description are correct and up-to-date
    - [x] `Draft: ` prefix is removed
    - [x] All temporary labels are removed
  
[l:1]: https://help.github.com/en/articles/about-pull-requests#draft-pull-requests
[l:2]: https://help.github.com/en/articles/reviewing-changes-in-pull-requests

### CODE REVIEW

**Comment by logist322** [comment]
## FCM
```
Add `Windows` support (#45)

- fix Dart imports
- implement some methods for screen sharing
```

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -235,8 +235,10 @@ jobs:
         include:
           - platform: android
             os: macos
-          - platform: linux
-            os: ubuntu
+#            TODO: Flutter requires libc6 of 2.34 version for some reason.
+#                  Uncomment when github will update ubuntu-latest to 22.04.
```
Comment (alexlapa) [inline .github/workflows/ci.yml:239]: @tyranron ,

Новая LTS'ка через два дня выходит, до гитхаба тоже, думаю, скоро доедет.

**Review by alexlapa — APPROVED** [review]

File: src/platform/dart/constraints.rs [hunk 2]
```diff
@@ -372,3 +386,19 @@ impl From<DeviceVideoTrackConstraints> for MediaTrackConstraints {
...
     }
 }
+
+impl From<DisplayVideoTrackConstraints> for MediaTrackConstraints {
+    fn from(from: DisplayVideoTrackConstraints) -> Self {
+        unsafe {
+            let optional =
+                DartHandle::new(constraints::new_video_constraints());
+            let mandatory =
+                DartHandle::new(constraints::new_video_constraints());
```
Comment (tyranron) [inline src/platform/dart/constraints.rs:396]: Is it OK that we don't use `from` input value here at all?

Comment (alexlapa) [reply src/platform/dart/constraints.rs:396]: @tyranron ,

Угу, у нас в `DisplayVideoTrackConstraints` лежит только `required: bool`. Т.е. мы сейчас не поддерживаем никакие констрейнты на screen-captured видео треки.

**Pull request merged**